UnityProjects/MRTKDevTemplate/Packages/manifest.json

```diff
@@ -1,7 +1,5 @@
 {
   "dependencies": {
-    "com.atteneder.gltfast": "https://github.com/atteneder/glTFast.git#v4.8.3",
```

Comment:
Let's replace this with the version on Unity's package manager: com.unity.cloud.gltfast

https://docs.unity3d.com/Packages/com.unity.cloud.gltfast@5.2/manual/

Reply:
This will likely involve adding a second asmdef define case for GLTFAST_PRESENT

Reply:
I believe to have attended the requested changes, please re-review.